File: class10/cpp/AP1010-BuscaNoVetor.cpp
#include <iostream>
int main() {
    int numeros[] = {10, 25, 33, 47, 52, 68, 71};
    int tamanho = sizeof(numeros)/sizeof(numeros[0]);
    int buscado;
    std::cout << "Digite o número a buscar: "; std::cin >> buscado;
    bool encontrado = false;
    for (int i = 0; i < tamanho; i++) {
        if (numeros[i] == buscado) {
            std::cout << "Encontrado na posição " << i << "!" << std::endl;
            encontrado = true;
            break;
        }
    }
    if (!encontrado) std::cout << "Número não encontrado no vetor." << std::endl;
    return 0;
}

File: class10/python/AP1010-BuscaNoVetor.py
# BuscaNoVetor — Aula 10
# Busca um valor em um vetor e retorna o índice

numeros = [10, 25, 33, 47, 52, 68, 71]
buscado = int(input("Digite o número a buscar: "))
encontrado = False
for i in range(len(numeros)):
    if numeros[i] == buscado:
        print(f"Encontrado na posição {i}!")
        encontrado = True
        break
if not encontrado:
    print("Número não encontrado no vetor.")
